recipes/unleash-client-cpp/all/patches/cmake-install.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/packaging/CMakeLists.txt b/packaging/CMakeLists.txt
```

Comment:
Don't need to patch it, avoid patching if you can. It will be better removing the cmake folder after install step, instead creating a patch.

Reply:
Done.